docs(activate): document auto-activation, allow/deny, and the BETA feature flag (DEV-89) (#4407)
flox/flox: cli/flox/doc/flox-activate-deny.md

---
title: FLOX-ACTIVATE-DENY
section: 1
header: "Flox User Manuals"
...

# NAME

flox-activate-deny - deny auto-activation for an environment

# SYNOPSIS

```text
flox [<general-options>] activate deny

>      [-d=<path> | -r=<owner>/<name> | -D]

Author: Same comment as for `flox-activate-allow` usage.

```suggestion
     [-d=<path>]
```

Author: Same fix as on `flox-activate-allow` — restricted to `-d`. https://github.com/flox/flox/commit/60c6479dc

```

# DESCRIPTION

```{.include}
./include/auto-activate-experimental.md
```

> Records that the selected environment must not be auto-activated.

Author: Same phrasing suggestion as `flox-activate-allow`.

```suggestion
Permits the selected environment to be auto-activated.
```

Author: Done — used the parallel wording "Prevents the selected environment from being auto-activated." (the suggestion pasted the *allow* text, which would invert the meaning here). https://github.com/flox/flox/commit/8237ac879
